src/Emulator/Main/Peripherals/CPU/ICpuSupportingGdb.cs

```diff
@@ -5,9 +5,39 @@
 // Full license text is available in 'licenses/MIT.txt'.
 //
 using System;
+using System.Collections.Generic;
 
 namespace Antmicro.Renode.Peripherals.CPU
 {
+    public struct IGBDRegister
+    {
+        public uint Number { get; }
+        public uint Size { get; }
+        public string Name { get; }
+        public string Type { get; }
+        public string Group { get; }
+
+        public IGBDRegister(uint number, uint size, string name, string type, string group) : this()
```

Comment:
According to our coding standards public constructor should go before properties